tests/test_bookmarks.py

```diff
@@ -0,0 +1,145 @@
+import tap_tester.connections as connections
+import tap_tester.menagerie   as menagerie
+import tap_tester.runner      as runner
+import unittest
+import simplejson
+
+from base import TestSquareBase
+from test_client import TestClient
+
+class TestSquareIncrementalReplication(TestSquareBase):
+    START_DATE_FORMAT = "%Y-%m-%dT00:00:00Z"
+    START_DATE = "2020-06-01T00:00:00Z"
+
+    def get_properties(self, original = True):
+        return_value = {
+            'start_date' : '2020-06-01T00:00:00Z',
+            'sandbox' : 'true'
+        }
+
+        if original:
+            return return_value
+
+        return_value['start_date'] = self.START_DATE
+        return return_value
```

Comment:
I saw this defined in base.py I would drop it from here

Reply:
This test needed a different start date and setting it on the TestCase didn't seem to do anything so we decided to override this function

Reply:
Ah gotcha, I handled this differently in other tap but I don't think it is necessarily a better way. This should be fine I think.